### PR TITLE
Search

### PR DESCRIPTION
**Pull request by anirudhpillai**
## What does this PR do?
Added Search API.

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?

## 📃 Link to PR updating documentation (if applicable)

https://github.com/uclapi/apiDocs/pull/24

## 🚨 Is this a breaking change for API clients?
No

## Anything else
- Added the people endpoint.

### CODE REVIEW

File: backend/uclapi/search/views.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    r = requests.get(url)
+
+    results = r.json()["response"]["resultPacket"]["results"][:20]
+
+    def serialize_person(person):
+        return {
+            "name": person["title"],
+            "department": person["metaData"].get("7", ""),
+            "email": person["metaData"].get("E", ""),
+            "status": person["metaData"].get("g", ""),
```
Comment (wilhelmklopp) [inline backend/uclapi/search/views.py]: Question about what this code does: If there's no `g` on `person["metaData"]` will "status" just become emptystring? Is that what's going on here?

**Comment by wilhelmklopp** [comment]
@anirudhpillai  Also would you be able to write a test for this view, maybe with some help from @jermenkoo ?

**Comment by ChristopherHammond13** [comment]
Super excited that we're doing this! :D

**Comment by jermenkoo** [comment]
Yeah, I can write some tests for this.

File: backend/uclapi/search/views.py (hunk 1 above)
Comment (anirudhpillai) [reply backend/uclapi/search/views.py]: Yes, just used this as sometimes the metadata fields were absent

**Comment by ChristopherHammond13** [comment]
Merging!

**Pull request merged**